OutOfSchool/OutOfSchool.WebApi.Tests/Common/CommunicationServiceTest.cs

```diff
@@ -165,4 +210,24 @@ private class TestResponse : IResponse
     {
         public string? Content { get; set; }
     }
+
+    private class TestError : IErrorResponse
```

Comment:
(Code style) Why `class` but not `record`? See the beauty:
`private record TestError(HttpStatusCode HttpStatusCode, string Message, string Content) : IErrorResponse {}`